rbac/management/group/definer.py

```diff
@@ -127,7 +137,6 @@ def add_roles(group, roles_or_role_ids, tenant, user=None):
     """Process list of roles and add them to the group."""
```

Comment:
I think we can remove the `@transaction.atomic` on add_roles and remove_roles now. They are both now necessarily called within outer atomic blocks so it's adding unnecessary savepoints I think.

Reply:
I removed it for `add_roles`, `remove_role`  functions.

`add_roles`, `remove_role` are not in transaction block for seed process. If I did not overlook anything It looks that whole seed process is not in transaction.

Do you agree with wrapping seed process into transaction in follow up PR? It seems beneficial.

Reply:
I think at least where `add_roles` and `remove_roles` are used in seeding, they are in atomic blocks:

```
def seed_group() -> Tuple[Group, Group]:
    """Create or update default group."""
    public_tenant = Tenant.objects.get(tenant_name="public")
    with transaction.atomic():
        ...
```

Reply:
oh, cool I overlooked it in that case, thanks!